js/constants/constants.js

```diff
@@ -106,7 +106,10 @@ module.exports = {
     AUTO_QUALITY: "Auto",
     SHARE: "Share",
     MORE_OPTIONS: "More Options",
-    SELECT_TO_UNMUTE: "SELECT TO UNMUTE"
+    SELECT_TO_UNMUTE: "SELECT TO UNMUTE",
```

Comment:
Can you change this to 'Select to unmute', so it's consistent with the other variables?

Reply:
This is the string we see for the unmute icon. The intent is for them to be all upper case.

Reply:
Ok, sounds good to me.